assertion/function/assertiontree/root_assertion_node.go

```diff
@@ -1175,8 +1175,16 @@ func (r *RootAssertionNode) isStable(expr ast.Expr) bool {
 			return true
 		}
 
-		// TODO: check for function names and user-declared constants
+		// package is considered a special case of ident to suppport selector expressions used to access stable
+		// expressions, such as constants declared in another package (e.g., pkg.Const)
+		if r.isPkgName(expr) {
+			return true
+		}
```

Comment:
Can we just combine the cases into a single `if` while we are at it? 😃

Reply:
Sure, done :)